src/app/home/home.module.ts

```diff
@@ -0,0 +1,14 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { IndexComponent } from './index/index.component';
+import { HomeRoutingModule } from './home-routing.module';
+
+
+@NgModule({
+  declarations: [IndexComponent],
+  imports: [
+    CommonModule, 
+    HomeRoutingModule
+  ]
+})
+export class HomeModule { }
```

Comment:
You don't need HomeModule, just a HomeComponent and route set in the main app-routing. You can rename this module to PageModule for example and then you will have Home, Contact Us, etc in the module "PageModule". If you have a single page, just create a component and include it in the app root.